### ISSUE TITLE
[CC] Leveling system bugged

### ISSUE THREAD

**Issue by Erik-Glover** [civilgamers/bug-reports#79]
Sometimes they dont level up even after being passted the progress threshhold. Add points, take points and purchase points also no longer work.

https://imgur.com/a/bZKbm1q

**Comment by Tiin57** [comment]
@dhog10 just remove?

**Comment by HydroCG** [comment]
No, we need it as it's coupled with mining and woodcutting. I have a really good dev branch in SP to fix this though so that can be done at some point.
